scripting/src/test/java/org/infinispan/scripting/ClusteredScriptingTest.java

```diff
@@ -86,26 +99,12 @@ public void testMapReduce() throws Exception {
       assertTrue(results.get("macbeth").equals(Double.valueOf(287)));
    }
 
-   private void loadData(Cache<String, String> cache, String fileName) throws IOException {
-      try (BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(this.getClass().getResourceAsStream(fileName)))) {
-         int chunkSize = 10;
-         int chunkId = 0;
-
-         CharBuffer cbuf = CharBuffer.allocate(1024 * chunkSize);
-         while (bufferedReader.read(cbuf) >= 0) {
-            Buffer buffer = cbuf.flip();
-            String textChunk = buffer.toString();
-            cache.put(fileName + (chunkId++), textChunk);
-            cbuf.clear();
-         }
-      }
-   }
-
-   private void loadScript(ScriptingManager scriptingManager, String fileName) throws IOException {
-      try (InputStream is = this.getClass().getResourceAsStream(fileName)) {
-         String script = TestingUtil.loadFileAsString(is);
-         scriptingManager.addScript(fileName.replaceAll("\\/", ""), script);
-      }
+   @Test(expectedExceptions = IllegalArgumentException.class, expectedExceptionsMessageRegExp = "^ISPN026008.*")
+   public void testOnlyReduceTaskRun() throws Exception {
+      ScriptingManager scriptingManager = getScriptingManager(manager(0));
+      Cache<String, String> cache = cache(0);
+      loadData(cache, "/macbeth.txt");
+      loadScript(scriptingManager, "/wordCountReducer.js");
+      CompletableFuture<Object> future = scriptingManager.runScript("wordCountReducer.js", new TaskContext().cache(cache));
```

Comment:
Right... but what do we want to do with `CompletableFuture<Object> future` reference? It has been assigned but it looks like it hasn't been used.